openapi/p4pa-workflow-hub.openapi.yaml

```
@@ -6,6 +6,36 @@ info:
 servers:
   - url: "http://localhost:8080/workflowhub"
 paths:
+  /PaymentIngestionWF/{ingestionFileId}:
```

let's follow the wf package partitioning
```suggestion
  /ingestion-flow/payments-reporting/{ingestionFlowFileId}:
```

---

Done